src/main/java/org/gridsuite/directory/server/dto/ElementAttributes.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.directory.server.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.gridsuite.directory.server.ElementType;
+
+import java.util.UUID;
+
+/**
+ * @author Nicolas Noir <nicolas.noir at rte-france.com>
+ */
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class ElementAttributes {
+
+    private UUID elementUuid;
```

Comment:
for /v1/directories/{directoryUuid}/add
We should only pass the directoryUuid so for the body json elementUuid should be deleted ? and server side generated  ?

Reply:
The idea is to add an element which already has an id. For example, we might want to add a study in the directory, which is already stored in a study database (using study-server services). We want to identify the study in the directory database with the same id it has in the study database. Anyway, setting a null elementUuid in ElementAttributes will generate a new uuid when adding the element in the database.